### DISCUSSION TITLE
Browser Window Size

### DISCUSSION THREAD

**Discussion by m3rlinux** [elabit/robotmk#183] [Q&A] [answered]
Hi! Is there a way to set the window size within the cmk plugin as same as running manual test from CMD?
I notice that within the plugin the window is smaller. I tryed by "Set Window Size" without success.

**Comment by simonmeggle** [comment, marked as answer]
Hi, 
I think it should not make a difference if a robot test was called manually from the cmd or with robotmk. 
For questions regarding RF libraries I want to recommend you the Slack channel or the Robot Framework user forum. There are subchannels for Selenium and Playwright as well.

Best regards, Simon
